vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1316,7 +1316,10 @@ def _prepare_inputs(
         # running prefills. This lets us set enforce_eager on the prefiller in
         # a P/D setup and still use CUDA graphs (enabled by this padding) on the
         # decoder.
-        allow_dp_padding = self.compilation_config.cudagraph_mode != CUDAGraphMode.NONE
+        allow_dp_padding = (
+            self.compilation_config.cudagraph_mode != CUDAGraphMode.NONE
+            or self.parallel_config.all2all_backend == "deepep_hybrid"
+        )
```

Comment:
Do the hybrid DeepEP kernels support CUDA Graphs? I'm seeing conflicting things in the PR